Mathlib/Tactic/Linter/Lint.lean

```diff
@@ -5,6 +5,9 @@ Authors: Floris van Doorn
 -/
 import Batteries.Tactic.Lint
 import Mathlib.Tactic.DeclarationNames
+-- Import this linter explicitly to ensure that
+-- this file has a valid copyright header and module docstring.
+import Mathlib.Tactic.Linter.Header
```

Comment:
Doesn't `DeclarationNames` already import `Header`?  This is redundant (though maybe it is not necessary to trim transitive imports here).

Reply:
Indeed, good spot - this is superfluous now.